spec/integration/operations_spec.rb

```diff
@@ -134,9 +134,20 @@ module Hawkular::Operations::RSpec
       end
 
       describe 'Operation/Operation' do
-        before(:all) do
-          @random_uuid = SecureRandom.uuid
-          @not_so_random_uuid = 'not_so_random_uuid'
+        let(:random_uuid) do
+          SecureRandom.uuid
+        end
+
+        let (:not_so_random_uuid) do
```

Comment:
Why do we need this? `let` declarations are memoized thorough the test case, so while it is random, it is consistent.